src/common/InputWithClearButton.js

```diff
@@ -32,7 +32,7 @@ export default class InputWithClearButton extends PureComponent<Props, State> {
     canBeCleared: false,
     text: '',
   };
-  textInputRef = React.createRef<typeof TextInput>();
+  textInputRef = React.createRef<React$ElementRef<typeof TextInput>>();
```

Comment:
A similar change should also happen at `Input` in the type of its prop that we pass this to.

(Though maybe not now after all; see previous comment.)